bin/steps/python

```diff
@@ -42,7 +42,7 @@ if [ ! "$SKIP_INSTALL" ]; then
 fi
 
 # If Pip isn't up to date:
-if [ "$FRESH_PYTHON" ] || [[ ! $(pip --version) == *$PIP_VERSION* ]]; then
+if [ "$FRESH_PYTHON" ] || [[ ! $(pip --version) == *$PIP_UPDATE* ]]; then
```

Comment:
turns out, pip completely fails to run if PIP_VERSION is set and isn't equal to what is installed.